src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -241,6 +242,7 @@ cudnnSetOpTensorDescriptor(opTensorDesc,
                     writer.indent--;
                     writer << "}\n";
                 }
+                //matrix vector
```

Comment:
Is is a standard to not put a space after `//` in a comment? I see it a lot lately. It looks better with a space there?

Reply:
Thanks, fixed.